tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

```diff
@@ -50,15 +50,47 @@
    * the implementation, but can in some cases result in longer time to
    * run the tests.
    * @param defaultTimeoutMillis default timeout to be used in all expect* methods
+   * @param defaultPollTimeoutMillis default amount of time to poll for events if {@code defaultTimeoutMillis} isn't
```

Comment:
Note that it only applies for errors as other signals are caught whenever they appear?
Refs https://github.com/reactive-streams/reactive-streams-jvm/pull/452/files#r278079420

Reply:
The rational was the `error` handling is an implementation detail and this polling can be used for all things that would otherwise have to block/wait the full `defaultTimeoutMillis` duration (e.g. if there is no async pre-emption). Although it is currently used for expected error timeouts due to internal implementation details, if the implementation was to change or be extended I think it is useful to have a "poll interval" and "upper bound" for these cases. However I'll defer to you if you want it more specialized.